packages/api-sdk/src/medical/models/demographics.ts

```diff
@@ -46,19 +47,20 @@ export type PersonalIdentifier = z.infer<typeof personalIdentifierSchema>;
 
 export const genderAtBirthSchema = z.enum(["F", "M"]);
 
-const phoneLength = 10;
 export const contactSchema = z
   .object({
     phone: z.coerce
       .string()
-      .transform(phone => stripNonNumericChars(phone))
-      .refine(phone => phone.length === phoneLength, {
-        message: `Phone must be a string consisting of ${phoneLength} numbers. For example: 4153245540`,
+      .transform(stripNonNumericChars)
```

Comment:
I still don't understand why we're allowing a user to store a non-numeric phone number...

Reply:
We're not, this step removes non numeric chars (thus, `transform()`) the input.

Reply:
Yea, realized it's a data transform.